api4/team.go

```diff
@@ -1045,6 +1153,7 @@ func inviteGuestsToChannels(c *Context, w http.ResponseWriter, r *http.Request)
 		}
```

Comment:
Same here.

Reply:
I've added all the attempted emails, and then all the error'd emails when failing gracefully.  https://github.com/mattermost/mattermost-server/pull/13967/commits/707fc3bcdfbc4d8bc4a3db1f710f96fd83311763